tests/integrationv2/providers.py

```diff
@@ -173,8 +173,7 @@ def setup_server(self):
         if self.options.protocol == Protocols.TLS13:
             cmd_line.append('--tls13')
 
-        if self.options.cipher is not None:
-            cmd_line.extend(['-c', 'test_all'])
+        cmd_line.extend(['-c', 'test_all'])
```

Comment:
`test_all` contains ciphers/policies for all TLS versions, for versions lesser than TLS13 we will have to use `test_all_tls12` instead. 

```
if self.options.protocol == Protocols.TLS13:
            cmd_line.extend(['-c', 'test_all'])
else: 
           cmd_line.extend(['-c', 'test_all_tls12'])

Reply:
I've added specific preferences, test_all_tls13, test_all_tls12, and test_all.